frontend/src/core/comments/components/Comment.css

```diff
@@ -5,7 +5,11 @@
 .comments-list .comment a {
     color: #7BC876;
     font-weight: 400;
-    white-space: nowrap;
+}
+
+.comments-list .comment a[href^="/contributors/"]:not(.comment-author) {
```

Comment:
This selector will also catch the avatar. Let's add `.container .content` after `.comment`.